build.js

```diff
@@ -120,32 +122,35 @@ await remove("./dist", {
 	force: true,
 });
 
+await copy("./src", "./dist", {
```

Comment:
Please bring the comment from below to here, and modify it according to the new context.